azext_iot/operations/hub.py

```diff
@@ -2609,7 +2609,7 @@ def conn_str_getter(hub):
                 if show_all
                 else conn_str_getter(hub)[0],
             }
-            for hub in hubs
+            for hub in hubs if hub.properties.state == 'Active'
```

Comment:
Is there a state enum in the SDK?

Reply:
Great question, not that I could find - I went ahead and added the enum type to our `shared` models